### PR TITLE
Utility docs

### PR DESCRIPTION
**Pull request by matt3224**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,124 +2,826 @@
...
+                    <hr>
+
+                    <p>This utility creates simple potentially helpful classes that you can use to set the background colour or text colour of any element. These are created based on the colour variables set out in the settings layer.</p>
+
+                    <pre class="line-numbers">
+                        <code class="language-markup">
+                            &lt;figure class=&quot;u-bgNoir&quot;&gt;
+                                &lt;img src=&quot;michael.jpg&quot; alt=&quot;Michael Scott&quot;&gt;
+                                &lt;figcaption class=&quot;u-colourSnow&quot;&gt;You don't know how high I can fly&lt;/figcaption&gt;
+                                &lt;/figure&gt;
```
Comment (darbymanning) [inline index.html]: Closing tag is indented too far

**Comment by darbymanning** [comment]
What is up with the commit history we're seeing here? Dating back to June 2016?

Added a comment. Also, needs a rebase... Otherwise LGTM

**Comment by matt3224** [comment]
Sorry mate what are you referring to? The complete history of the project?

Also what is lgtm?

**Comment by darbymanning** [comment]
LGTM - looks good to me. 

I’m referring to the commit history that’s on this PR. Seems to be showing historic commits unrelated to this work?

**Pull request merged**